modules/runner/src/integrationTest/java/org/apache/ignite/internal/runner/app/ItIgniteNodeRestartTest.java

```diff
@@ -413,7 +415,8 @@ private PartialNode startPartialNode(
                 schemaSyncService,
                 catalogManager,
                 new HybridTimestampTracker(),
-                placementDriverManager.placementDriver()
+                placementDriverManager.placementDriver(),
+                () -> mock(IgniteSql.class)
```

Comment:
As I see the test hadn't use mock before the patch. Why you decided to mix different approaches?